### PR TITLE
APB-850 Onboarding - [FE] Resume Savings Account-i application

### PR DESCRIPTION
**Pull request by jaypoojara101digital**
APB-850 Onboarding - [FE] Resume Savings Account-i application

Link to Jira ticket: https://acsdigital.atlassian.net/browse/APB-850

### CODE REVIEW

File: src/components/adb-lite-home-screen/components/functionality-list.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+type FunctionalityList = {
+  items: Item[];
+  onSelectItem: (id: string) => void;
+  disabled: boolean;
+};
+
+const FunctionalityList = ({ items, onSelectItem, disabled }: FunctionalityList) => {
+  return (
+    <FlatList
+      style={{ marginVertical: 20 }}
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/components/functionality-list.tsx:20]: inline style

File: src/components/adb-lite-home-screen/components/product-carousel-view.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+const ProductCarouselView: React.FC<IProductCarouselView> = (
+    props: IProductCarouselView
+) => {
+    const { title = '', subTitle = '', onSelect, disabled } = props;
+
+    return (
+        <TouchableOpacity onPress={onSelect} style={styles.container} disabled={disabled}>
+            <View style={{ alignItems: 'center', position: 'absolute', top: 10, left: 10, width: 111, paddingVertical: 10, justifyContent: 'center', backgroundColor: "#8E8E93", borderRadius: 40 }}>
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/components/product-carousel-view.tsx:20]: inline style

File: src/components/adb-lite-home-screen/components/product-carousel-view.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                <Text style={styles.title}>{title}</Text>
+            </View>
+            <ImageIcon size={20} color={colors.white} />
+            <View style={{ alignSelf: 'flex-end', position: 'absolute', bottom: 5, left: 12 }}>
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/components/product-carousel-view.tsx:24]: inline style

File: src/components/adb-lite-home-screen/index.tsx [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+    if (selectedAccount) {
+      onSelectAccount(selectedAccount);
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.row}>
+        <TouchableOpacity style={styles.logoutBtn} onPress={onNavigateToSetting}>
+          <ImageIcon size={30} color={'#FFFFFF'} />
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/index.tsx:73]: use theme colors

File: src/components/adb-lite-home-screen/index.tsx [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+          <RefreshControl
+            refreshing={isLoadingGetWallets}
+            onRefresh={getWallets}
+          />
+        }
+      >
+        <View style={styles.content}>
+          <TouchableOpacity onPress={onOpenAccount} style={styles.openSavingAccountContainer}>
+            <DollarIcon height={22} width={22} />
+            <View style={{ width: '80%' }}>
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/index.tsx:89]: inline style

File: src/components/adb-member-plus-dashboard/index.tsx [hunk 6]
```diff
@@ -326,6 +326,10 @@ const styles = StyleSheet.create({
     fontSize: 14,
     fontFamily: fonts.regular,
   },
+  openSavingAccountContainer: { height: 76, borderRadius: 12, width: '100%', padding: 10, backgroundColor: '#8E8E93', flexDirection: 'row', alignItems: 'center', marginVertical: 10, justifyContent: 'space-around' },
```
Comment (sharmal33) [inline src/components/adb-member-plus-dashboard/index.tsx:329]: use theme colors

File: src/components/adb-member-plus-dashboard/index.tsx [hunk 7]
```diff
@@ -326,6 +326,10 @@ const styles = StyleSheet.create({
...
+  savingAccountLabel: {
+    color: '#FCFCFC', fontSize: 10, fontFamily: fonts.regular
```
Comment (sharmal33) [inline src/components/adb-member-plus-dashboard/index.tsx:331]: use theme colors

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-lite-home-screen/components/functionality-list.tsx (hunk 1 above)
Comment (jaypoojara101digital) [reply src/components/adb-lite-home-screen/components/functionality-list.tsx]: Resolved

File: src/components/adb-lite-home-screen/components/product-carousel-view.tsx (hunk 2 above)
Comment (jaypoojara101digital) [reply src/components/adb-lite-home-screen/components/product-carousel-view.tsx]: Resolved

File: src/components/adb-member-plus-dashboard/index.tsx (hunk 6 above)
Comment (jaypoojara101digital) [reply src/components/adb-member-plus-dashboard/index.tsx]: Removed code it's not required here now.

File: src/components/adb-member-plus-dashboard/index.tsx (hunk 7 above)
Comment (jaypoojara101digital) [reply src/components/adb-member-plus-dashboard/index.tsx]: Removed code it's not required here now.

File: src/components/adb-lite-home-screen/index.tsx [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+
+          <View style={styles.mermberShipContainer}>
+            <Text style={styles.label16}>{i18n.t('lite_home.membership_details') ?? 'Membership details'}</Text>
+            <View style={styles.centerRow}>
+              <Text style={styles.label10}>{i18n.t('lite_home.my_point') ?? 'My points'}</Text>
+              <View style={styles.width4} />
+              <InfoIcon height={16} width={16} color={colors.primaryBlack} />
+            </View>
+            <Text style={styles.label20}>1000</Text>
+            <Text style={styles.label10}>{`{200 pts} will be expired on {Mar 2023}`}</Text>
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/index.tsx:106]: why this one is hardcoded ?

File: src/components/adb-lite-home-screen/index.tsx [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+            <View style={styles.height12} />
+            <BarcodeIcon height={64} width={300} />
+            <View style={styles.rowSpaceBetween}>
+              <Text style={styles.label}>{`ID: 2022 0302 1992 1120`}</Text>
+              <Text style={styles.label}>{i18n.t('lite_home.expiry') ?? 'Expiry'} 10/26</Text>
```
Comment (sharmal33) [inline src/components/adb-lite-home-screen/index.tsx:111]: hardcoded

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-lite-home-screen/index.tsx (hunk 8 above)
Comment (jaypoojara101digital) [reply src/components/adb-lite-home-screen/index.tsx:106]: This data should be coming from the api back end so. I have just added screen design part. @sharmal33 for showing the resume application popup on that screen.

File: src/components/adb-lite-home-screen/index.tsx (hunk 9 above)
Comment (jaypoojara101digital) [reply src/components/adb-lite-home-screen/index.tsx:111]: This data should be coming from the api back end so. I have just added screen design part. @sharmal33 for showing the resume application popup on that screen.

**Pull request merged**